native/common/jp_field.cpp

```diff
@@ -17,113 +17,75 @@
 #include "jpype.h"
 #include "jp_field.h"
 
-JPField::JPField(JPClass* clazz, jobject fld) : m_Field(fld)
+JPField::JPField(JPJavaFrame& frame,
+		JPClass* cls,
+		const string& name,
+		jobject field,
+		jfieldID fid,
+		JPClass* fieldType,
+		jint modifiers)
+: m_Field(frame, field)
 {
-	JPJavaFrame frame;
-	//	JP_TRACE_IN("JPField::JPField1");
-	m_Class = clazz;
-	m_Name = JPJni::getMemberName(fld);
-	m_IsStatic = JPJni::isMemberStatic(fld);
-	m_IsFinal = JPJni::isMemberFinal(fld);
-	m_FieldID = frame.FromReflectedField(fld);
-	m_Type = JPJni::getFieldType(m_Field.get());
-	m_TypeCache = NULL;
-
-	//	JP_TRACE_OUT
+	this->m_Class = cls;
```

Comment:
the self reference code style is newly introdcued in this class. It is not mandatory and makes the code harder to read IMO. BTW what happened to good old initializer lists?

Reply:
Removing it.  It is a stupid editor trick I use to help with auto complete while typing fresh code.  It should have been removed during my clean up pass.

Reply:
Removing it.  It is a stupid editor trick I use to help with auto complete while typing fresh code.  It should have been removed during my clean up pass.

Reply:
Fixed in 19 places.